wazo_confd_client/commands/emails.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from wazo_lib_rest_client import HTTPCommand
+from wazo_confd_client.util import url_join
+
+
+class EmailsCommand(HTTPCommand):
+
+    headers = {'Accept': 'application/json'}
+
+    def get(self):
+        url = url_join('emails')
+        r = self.session.get(url, headers=self.headers)
+
+        return r.json()
+
+    def update(self, body):
+        url = url_join('emails')
+        self.session.put(url, json=body, headers=self.headers)
```

Comment:
Any reason to not use RESTCommand?
ex: https://github.com/wazo-platform/wazo-confd-client/blob/master/wazo_confd_client/commands/confbridge_wazo_default_bridge.py